mirgecom/boundary.py

```diff
@@ -53,20 +53,20 @@ def __init__(self, userfunc):
         Parameters
         ----------
         userfunc
-            User function must take two parameters: time and nodal
-            coordinates, and produce the solution at each node.
+            User function that prescribes the solution values on the exterior
+            of the boundary. The given user function (*userfunc*) must take at
+            least one parameter that specifies the coordinates at which to prescribe
+            the solution.
         """
         self._userfunc = userfunc
 
-    def boundary_pair(
-            self, discr, q, btag, eos, t=0.0
-    ):
+    def boundary_pair(self, discr, q, btag, *args, **kwargs):
```

Comment:
AFAICT `*args get swallowed silently. That shouldn't happen IMO.

Reply:
I thought I had fixed these.  Must have missed one.